k8s/chainctl

```diff
@@ -317,6 +331,33 @@ case "$1" in
     certbot -c certbot.ini -d $DOMAIN --manual --preferred-challenges dns certonly
     ;;
 
+  ssh)
+    select_gcloud_config
+    select-cluster
+    kubectl exec -it `kubectl get pods | grep -v NAME | awk '{print$1}'` -- /bin/bash
+    ;;
+
+  attach)
+    select_gcloud_config
+    select-cluster
+    kubectl attach `kubectl get pods | grep -v NAME | awk '{print$1}'`
```

Comment:
This might be worth putting into a function? I'd probably also add `--no-headers` to `kubectl`